src/SimplePDPService.sol

```diff
@@ -171,16 +171,16 @@ contract SimplePDPService is PDPListener, PDPRecordKeeper, Initializable, UUPSUp
     // it also checks that proofs are not late and emits a fault record if so
     function possessionProven(uint256 proofSetId, uint256 challengedLeafCount, uint256 seed, uint256 challengeCount) external onlyPDPVerifier {
         receiveProofSetEvent(proofSetId, OperationType.PROVE_POSSESSION, abi.encode(challengedLeafCount, seed, challengeCount));
-        if (provenThisPeriod[proofSetId]) { 
-            revert("Only one proof of possession allowed per proving period. Open a new proving period."); 
```

Comment:
Hmm these extra spacing changes are frustrating. Sorry. Will dive deep into my editor settings today to see why this is happening.

Reply:
At this point I'm getting used to it so don't worry too much about it

Reply:
@ZenGround0 Found the root cause in my editor and fixed it. I'll add a formatter to our CI so we can stop worrying about it.